spec/03-plugins/23-rate-limiting/06-shorthand_fields_spec.lua

```diff
@@ -1,11 +1,10 @@
 local helpers = require "spec.helpers"
-local utils = require "kong.tools.utils"
 local cjson = require "cjson"
 
 
 describe("Plugin: rate-limiting (shorthand fields)", function()
   local bp, route, admin_client
-  local plugin_id = utils.uuid()
+  local plugin_id = require("kong.tools.uuid").uuid()
```

Comment:
It may conflict with https://github.com/Kong/kong/pull/12952

Reply:
modified